software/database/pom.xml

```diff
@@ -144,11 +144,24 @@
         </dependency>
 
         <dependency>
+            <groupId>${jclouds.groupId}.provider</groupId>
```

Comment:
Maven test dependencies are not transitive. Need to be explicit about test dependencies.